kclvm/tools/src/vet/expr_builder.rs

```diff
@@ -135,7 +135,7 @@ impl ExprGenerator<serde_yaml::Value> for ExprBuilder {
 
                 for (k, v) in j_map.iter() {
                     let k = self
-                        .generate(k, schema_name)
+                        .generate(k, &None)
```

Comment:
U can leave a comment. `The configuration builder already in the schema no longer needs a schema name`

Reply:
fixed